### PR TITLE
Add Bio for Cedd Burge

### PR DESCRIPTION
**Pull request by ceddlyburge**
The readme states that I should add a Bio for each track, which I have done. I focus on the Elm track however, which I rarely get to the end of, and this has meant that I haven't done any mentoring on the C# track. I'm not sure if this is a problem or not.

### CODE REVIEW

File: mentors/csharp/c.json [hunk 1]
```diff
@@ -22,5 +22,13 @@
     "link_url": "https://medium.com/@chaisstar19",
     "avatar_url": "https://miro.medium.com/fit/c/128/128/2*yx4DpH7FrYM8PucaZXNrVw.jpeg",
     "bio": "I use C# every day at work for building web application. And I believe, that the best way to learn something new is to teach someone"
+  },
+  {
+    "github_username": "ceddlyburge",
+    "name": "Cedd Burge",
+    "link_text": "Blog Posts",
+    "link_url": "https://medium.com/@cuddlyburger",
+    "avatar_url": https://github.com/ceddlyburge.png,
```
Comment (rpottsoh) [inline mentors/csharp/c.json]: url need to be enclosed in double quotes `"`.
```suggestion
    "avatar_url": "https://github.com/ceddlyburge.png",
```

File: mentors/elm/c.json [hunk 2]
```diff
@@ -6,5 +6,13 @@
     "link_url": "",
     "avatar_url": null,
     "bio": "I am drawn to Elm because it is a safe, reliable, and refactorable alternative to building webapps. It's an approachable language that not only supports the dynamic nature of greenfield projects, but it can also be adopted in small steps within legacy apps implemented in other front-end solutions."
+  },
+  {
+    "github_username": "ceddlyburge",
+    "name": "Cedd Burge",
+    "link_text": "Blog Posts",
+    "link_url": "https://medium.com/@cuddlyburger",
+    "avatar_url": https://github.com/ceddlyburge.png,
```
Comment (rpottsoh) [inline mentors/elm/c.json]: url need to be enclosed in double quotes `"`.
```suggestion
    "avatar_url": "https://github.com/ceddlyburge.png",
```

File: mentors/typescript/c.json [hunk 3]
```diff
@@ -5,4 +5,13 @@
...
     "avatar_url": null,
     "bio": "Using typescript professionally on projects, I love to network and help"
-  }]
+  },
+  {
+    "github_username": "ceddlyburge",
+    "name": "Cedd Burge",
+    "link_text": "Blog Posts",
+    "link_url": "https://medium.com/@cuddlyburger",
+    "avatar_url": https://github.com/ceddlyburge.png,
```
Comment (rpottsoh) [inline mentors/typescript/c.json]: url need to be enclosed in double quotes `"`.
```suggestion
    "avatar_url": "https://github.com/ceddlyburge.png",
```

File: mentors/typescript/c.json [hunk 4]
```diff
@@ -5,4 +5,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "Using typescript professionally on projects, I love to network and help"
-  }]
```
Comment (rpottsoh) [inline mentors/typescript/c.json]: Thanks for fixing  this.

**Comment by ceddlyburge** [comment]
Oooops, invalid json, silly me. Thanks for the suggestions, I've comitted them.

Cheers

Cedd

**Pull request merged**